x/emissions/module/rewards/rewards_internal.go

```diff
@@ -445,6 +445,13 @@ func GetAllReputersOutput(
 	coefficients := make([]alloraMath.Dec, len(initialCoefficients))
 	copy(coefficients, initialCoefficients)
 
+	// Check if there's only one reputer and the listening coefficient is zero
+	// If so, set the listening coefficient to 0.5
+	// This is to ensure there's no bugs in the scores calculation
+	if len(coefficients) == 1 && coefficients[0].IsZero() {
+		coefficients[0] = alloraMath.MustNewDecFromString("0.5")
```

Comment:
mrrr hardcoded constants mrrrr

do we not have a global param for this

Reply:
This covers in the 1-reputer case, but is it possible that we have coefficiencts = zero in a multi-reputer scenario?